### PR TITLE
Add topic aggregation query

### PR DESCRIPTION
**Pull request by rahulPalliyalil**
### What

This pr contains the following changes:

- Add topic aggregation query
- Rename doccounts to contentTypeCounts

### How to review

Check if the changes make sense

### Who can review

Anyone except me

### CODE REVIEW

File: models/elasticsearch.go [hunk 1]
```diff
@@ -94,6 +95,11 @@ type ESHighlight struct {
 	DescriptionDatasetID []*string `json:"description.datasetId"`
 }
 
+type Topic struct {
```
Comment (justinpjose) [inline models/elasticsearch.go:98]: Both `Topic` and `ContentType` struct are the same. We can make a common struct called `FilterCount` which can then be used for `Topics` and `ContentTypes`

File: query/templates/search/v710/countQuery.tmpl [hunk 2]
```diff
@@ -19,11 +19,11 @@
    },
   "size": 0,
   "aggregations": {
-  		"docCounts": {
```
Comment (justinpjose) [inline query/templates/search/v710/countQuery.tmpl:22]: (question) I am assuming that the content type counts were referred as `docCounts` before? Or is `docCounts` something else?

**Comment by rahulPalliyalil** [comment]
> I tested this locally and I can see the `topics` field returned at the root level. Unfortunately, I only have sandbox data where there are no searchable documents with topics. Are you able to demo this with mock data by any chance?

yes I can show a similar demo which i showed to Nathan this morning if you are interested

File: models/elasticsearch.go (hunk 1 above)
Comment (rahulPalliyalil) [reply models/elasticsearch.go]: Done

**Review by justinpjose — APPROVED** [review]

**Pull request merged**